tlatools/org.lamport.tlatools/src/tla2sany/configuration/ConfigConstants.java

```diff
@@ -260,24 +262,72 @@ public interface ConfigConstants {
    * canonical way.
    */
   String[][] OperatorSynonyms = new String[][] {
-    new String[] {"\\lnot", "~", "\\neg"},
-    new String[] {"\\land", "/\\"},
-    new String[] {"\\lor", "\\/"},
-    new String[] {"\\equiv", "<=>"},
-    new String[] {"/=", "#"},
-    new String[] {"\\leq", "<=", "=<"},
-    new String[] {"\\geq", ">="},
-    // The \mod synonym doesn't seem to be recognized in other parts of the
-    // code but keeping it here because it was in the original config.
-    new String[] {"%", "\\mod"},
-    new String[] {"\\times", "\\X"},
-    new String[] {"\\intersect", "\\cap"},
-    new String[] {"\\union", "\\cup"},
-    new String[] {"\\o", "\\circ"},
-    new String[] {"\\oplus", "(+)"},
-    new String[] {"\\ominus", "(-)"},
-    new String[] {"\\odot", "(.)"},
-    new String[] {"\\oslash", "(/)"},
-    new String[] {"\\otimes", "(\\X)"},
+    new String[] {"\\lnot", "~", "\\neg", "¬"},
+    new String[] {"[]", "□"},
+    new String[] {"<>", "◇"},
```

Comment:
This PR looks great, but I will insist on one small change:

Because we now have non-ASCII characters in the Java code, we should explicitly specify the source encoding in all of the `<javac>` Ant tasks. [There is an `encoding` option for this.](https://ant.apache.org/manual/Tasks/javac.html) I am guessing it should be UTF-8 (although I haven't actually checked what encoding you used for these characters in the Java code).

A couple of stray notes ---

 - While UTF-8 seems to be winning the unicode encoding war, it is still far from a universal default. ISO-8859-1 and UTF-16 are also popular. (Fun fact! ISO-8859-1 is the official default encoding for HTTP, not UTF-8, despite what lots of code seems to believe---leading to many corrupted web pages that are written by server software using UTF-8 and reinterpreted by standards-compliant browsers as ISO-8859-1. This seems to be improving with time, but I find it is still a frequent problem.)
 - To my surprise, the Java Language Specification is [strangely silent](https://docs.oracle.com/javase/specs/jls/se17/html/jls-3.html) on the subject of Java's default encoding. Since it only mentions ASCII, I am guessing it only _promises_ ASCII compatibility. Fortunately javac has an encoding argument to let us be explicit, and it does support many common encodings.
 - To my continued surprise, the major build systems (including Ant) are also silent on the subject of default Java source encoding. Again, I think we can only assume ASCII compatibility unless we set it to something else.

Reply:
Also, when we document unicode support for TLA+ end users, we should be explicit about what character encoding to use for TLA+ code. Previously it was ASCII, and now it has been expanded to UTF-8. Unlike Java and HTTP, there are no legal alternative encodings for TLA+ (which I think is good).